### PR TITLE
Add quickstart for Windows VM with ssh

### PR DESCRIPTION
**Pull request by tfenster**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add a bicep-based quickstart template to create a Windows Server VM with SSH installed and key-based authentication configured

### CODE REVIEW

**Comment by tfenster** [comment]
@bmoore-msft this is failing because of missing quota. Is there any recommendation what to use as default for a VM to avoid that problem? I couldn't find anything in the contribution or best practice guides

File: quickstarts/microsoft.compute/vm-windows-ssh/main.bicep [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    ipConfigurations: [
+      {
+        name: 'ipConfig1'
+        properties: {
+          privateIPAllocationMethod: 'Dynamic'
+          publicIPAddress: {
+            id: publicIP.id
+          }
+          subnet: {
+            id: resourceId('Microsoft.Network/virtualNetworks/subnets', 'virtualNetwork', 'subnet')
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/vm-windows-ssh/main.bicep:159]: use ```virtualNetwork.properties.subnets[0].id``` and then remove the dependsOn below

File: quickstarts/microsoft.compute/vm-windows-ssh/main.bicep [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+      networkInterfaces: [
+        {
+          id: nic.id
+        }
+      ]
+    }
+    diagnosticsProfile: {
+      bootDiagnostics: {
+        enabled: true
+        storageUri: reference(resourceId('Microsoft.Storage/storageAccounts/', storagename)).primaryEndpoints.blob
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/vm-windows-ssh/main.bicep:231]: use ```storageAccount.properties.primaryEndpoints.blob``` and remove dependsOn below

File: quickstarts/microsoft.compute/vm-windows-ssh/main.bicep [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+param adminUser string
+
+@secure()
+param adminPassword string = newGuid()
+
+@description('Your public SSH key')
+param publicSshKey string
+
+@description('VM size for VM')
+param vmsize string = 'Standard_D4_v5'
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/vm-windows-ssh/main.bicep:17]: I've submitted a request for quota for V5 series, until that goes through you can use v3

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by tfenster** [comment]
@bmoore-msft seems to me like the script can't be downloaded from the artifactsLocation here https://github.com/Azure/azure-quickstart-templates/blob/7f2b9d61708db9892b17c5a51277ed0b047df15b/quickstarts/microsoft.compute/vm-windows-ssh/azuredeploy.json#L106. But this should work, right? Or am I missing something?

**Comment by bmoore-msft** [comment]
I think you need to move the ```initialize.ps1``` to the root folder.  When the windows custom script extension downloads something from Azure storage it doesn't preserve the path.  During CI, we stage the files in Azure storage (they aren't pulled from github).

So move the script to the root and then update the code to reference the script from the root.

**Comment by tfenster** [comment]
@bmoore-msft Great, thanks for that hint, now it works! 

As you answered so quickly on this one, may I ask whether anything is wrong with #12095 as you gave no feedback there yet?

**Comment by bmoore-msft** [comment]
I haven't looked at 12095 yet ;)

**Comment by tfenster** [comment]
> I haven't looked at 12095 yet ;)

That certainly is a valid reason :)

**Pull request merged**